tools/matRad_compareDose.m

```diff
@@ -0,0 +1,166 @@
+%   cube1:         dose cube 1 as an M x N x O array
+%   cube2:         dose cube 2 as an M x N x O array
+%   ct:            plane view (coronal=1,sagittal=2,axial=3)
+%   cst:           list of interesting volumes inside the patient
+%   criteria:      [1x2] vector (optional) specifying the distance to agreement
+%                  criterion; first element is percentage difference,
+%                  second element is distance [mm], default [3 3]
+%   n:             number of interpolations (optional). there will be 2^n-1 
+%                  interpolation points. The maximum suggested value is 3.
+%                  default n=0
+%   localglobal:   parameter to choose between 'global' and 'local' 
+%                  normalization (optional)
+%
+%
+% output 
+%
+%   gammaCube:      result of gamma index calculation
+%   gammaPassRate:  rate of voxels passing the specified gamma criterion 
+%                   evaluated for every structure listed in 'cst'.
+%                   note that only voxels exceeding the dose threshold are
+%                   considered.
+%   hfig:           Figure handle struct for all 3 figures and subplots,
+%                   indexed by plane names
+%
+% References gamma analysis:
+%   [1]  http://www.ncbi.nlm.nih.gov/pubmed/9608475
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2015 the matRad development team. 
+% 
+% This file is part of the matRad project. It is subject to the license 
+% terms in the LICENSE file found in the top-level directory of this 
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part 
+% of the matRad project, including this file, may be copied, modified, 
+% propagated, or distributed except according to the terms contained in the 
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+%% check if cubes consistent
+if ~isequal(size(cube1),size(cube2))
+   error('dose cubes must be the same size\n'); 
+end
+
+if ~exist('localglobal','var')
+    localglobal = 'global';
+end
+if ~exist('n','var')
+    n = 0;
+end
+if ~exist('criteria','var')
+    criteria = [3 3];
+end
+%%
+resolution = [ct.resolution.x ct.resolution.y ct.resolution.z];
+
+%% Calculate iso-center slices
+isoCenter=matRad_getIsoCenter(cst,ct,0);
+slicename ={round(isoCenter(2)./resolution(2)),round(isoCenter(1)./resolution(1)),round(isoCenter(3)./resolution(3))};
+doseWindowCube2 = [0 max([cube2(:); cube2(:)])];
+doseWindowCube1 = [0 max([cube1(:); cube1(:)])];
+planename={'coronal','sagittal','axial'};
+
+%% Get the gamma cube
+disp('Calculating gamma index cube...');
+if exist('criteria','var')
+    relDoseThreshold = criteria(1); % in [%]
+    dist2AgreeMm     = criteria(2); % in [mm]
+else
+    dist2AgreeMm     = 3; % in [mm]
+    relDoseThreshold = 3; % in [%]
+end
+
+[gammaCube,gammaPassRate] = matRad_gammaIndex(cube1,cube2,resolution,criteria,[],n,localglobal,cst);
+set(gcf,'visible','off')
```

Comment:
resolve bug in gamma index calculation. maybe figure is created when colormap is defined?